cg/cli/workflow/commands.py

```diff
@@ -112,9 +119,32 @@ def store_available(context: click.Context, dry_run: bool) -> None:
         raise click.Abort
 
 
+@click.command("clean-rsync-dirs")
+@OPTION_YES
+@OPTION_DRY
+@ARGUMENT_BEFORE_STR
+@click.pass_obj
+def clean_rsync_dirs(context: CGConfig, before_str: str, dry_run: bool, yes: bool) -> None:
+    """Remove deliver workflow commands"""
+
+    rsync_api: RsyncAPI = RsyncAPI(config=context)
+
+    before = parse_date(before_str)
+
+    for process in rsync_api.rsync_processes:
+        if rsync_api.process_ready_to_clean(before=before, process=process):
+            if dry_run:
+                LOG.info(f"Would have removed {process}")
```

Comment:
Since they are multiple processes we would interrupt the looping if we return success here. I would want it to show all the directories that exists and which ones it would delete.

Reply:
True that! But then you should move the dry-run if to be within the confirmation block so the only thing not being done is the `shutil.rmtree(process, ignore_errors=True)`

Reply:
maybe pass a continue then?